### PR TITLE
Insert bulk db transaction

### PR DESCRIPTION
**Pull request by ribeirojose**

### CODE REVIEW

**Comment by ribeirojose** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 14
           

Complexity increasing per file
==============================
- old_timesheet_migration/timesheet_migration.py  5
- update_member_clt.py  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/ribeirojose/clockify-integration/pullRequest?prid=4468659&bid=15091153)

File: Pipfile [hunk 1]
```diff
@@ -14,8 +14,10 @@ orator = "*"
 psycopg2-binary = "*"
 requests = "*"
 sendgrid = "*"
-chalice = "*"
 boto3 = "*"
+psycopg2 = "*"
```
Comment (lukasab) [inline Pipfile]: delete this?

File: orator_development.py [hunk 2]
```diff
@@ -1,10 +1,17 @@
...
-        'user': 'neo',
-        'password': 'neoempresarial',
-        'prefix': ''
+    "production": {
+        "driver": "postgres",
+        "host": DB_HOSTNAME,
+        "database": DB_NAME,
+        "user": DB_USERNAME,
+        "password": DB_PASSWORD,
+        "prefix": "",
```
Comment (lukasab) [inline orator_development.py]: Create a new file for this. Maybe `orator_production.py`

File: old_timesheet_migration/timesheet_migration.py [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+    empty_time_entries = pd.DataFrame(columns=header_time_entries)
+
+    timesheets_path = join(
+        dirname(dirname(abspath(__file__))), "old_timesheet_migration", "timesheets"
+    )
+    os.chdir(timesheets_path)
+
+    for timesheet in glob.glob("*xlsx"):
+        year_semester = timesheet.split("_")[3].split(".")[0][:5]
+        path = timesheets_path + "/" + timesheet
```
Comment (lukasab) [inline old_timesheet_migration/timesheet_migration.py]: We should use os.join
```suggestion
        path = os.path.join(timesheets_path, timesheet)
```
This makes it usable for Windows, Linux, and Mac.

File: old_timesheet_migration/timesheet_migration.py [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+        clean_timesheet = read_timesheet(year_semester, path)
+        time_entries = create_time_entries(empty_time_entries, clean_timesheet)
+        time_entries_with_ids = fill_ids(time_entries)
+
+        # Save time_entries in csv file at the same directory that has the timesheets
+        time_entries_with_ids.to_csv(
+            timesheets_path + "/" + "time_entries" + year_semester + ".csv"
```
Comment (lukasab) [inline old_timesheet_migration/timesheet_migration.py]:
```suggestion
            os.path.join(timesheets_path, "time_entries" + year_semester + ".csv")
```

File: orator_development.py (hunk 2 above)
Comment (ribeirojose) [reply orator_development.py]: I let them all be in the same orator.py. Tell me what you think.

**Pull request merged**